packages/engine/src/actions/validate.js

```diff
@@ -16,7 +16,7 @@
 
 import getBlockMatcher from '../getBlockMatcher.js';
 
-async function Validate({ context, params }) {
+const validate = ({ context, params }) => {
   const validationErrors = context.RootBlocks.validate(getBlockMatcher(params));
   if (validationErrors.length > 0) {
```

Comment:
@Gervwyk Should the error be thrown by the action or the method? I am not sure on this one

Reply:
Also not sure. I think it is an error, so the method should throw an error. The action can always catch and handle the implementation etc.